test/suite/helper.go

```diff
@@ -114,3 +114,73 @@ func FloatMapMessage(topic, messageType, service string, value map[string]float6
 		),
 	}
 }
+
```

Comment:
If you're making builder, make one, and add a method Bool/String/Etc in it.

Reply:
Good idea!

Reply:
Fixed.